modules/react-components/src/components/tab/resource-tab/resource-tab.tsx

```diff
@@ -113,24 +121,57 @@ export const ResourceTab: FunctionComponent<ResourceTabPropsInterface> & Resourc
         setActiveIndex(activeIndex);
     };
 
+    if(isLoading === false) {
+        return (
+            <Tab
+                onTabChange={ (e: MouseEvent<HTMLDivElement>, data: TabProps) => {
+                    handleTabChange(e, data.activeIndex);
+                    onTabChange && typeof onTabChange === "function" && onTabChange(e, data);
+                } }
+                className={ classes }
+                menu={ {
+                    attached,
+                    pointing,
+                    secondary
+                } }
+                panes={ panes }
+                activeIndex={ activeIndex }
+                data-componentid={ componentId }
+                { ...rest }
+            />
+        );
+    }
+
     return (
-        <Tab
-            onTabChange={ (e: MouseEvent<HTMLDivElement>, data: TabProps ) => {
-                handleTabChange(e, data.activeIndex);
-                onTabChange && typeof onTabChange === "function" && onTabChange(e, data);
-            } }
-            className={ classes }
-            menu={ {
-                attached,
-                pointing,
-                secondary
-            } }
-            panes={ panes }
-            activeIndex={ activeIndex }
-            data-componentid={ componentId }
-            data-testid={ testId }
-            { ...rest }
-        />
+        <>
+            <Card.Group style={ { marginBottom: "3rem" } } >
+                { [ ...Array(tabCount) ].map(() => {
```

Comment:
Since the tabCount is optional and can be undefined, the output array of this can be `[undefined]`. Will need to have a undefined check

Reply:
A default prop for `tabCount` is set in this component. Do we still need to have the check?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3589/commits/b0288c818f91702f8b8f379929092552c583055f